dataPipelines/gc_thumbnails/cli.py

```diff
@@ -34,15 +34,25 @@ def cli():
     type=click.Path(dir_okay=True, resolve_path=True),
     default=os.path.abspath('.')
 )
+@click.option(
+    '-m',
+    '--max-threads',
```

Comment:
better way to generalize this param would be to call it "workers" and not talk about processes/threads at all. So `--max-workers`